docs/use_cases/batch/part1.md

```diff
@@ -559,7 +614,31 @@ group by
 order by
 	revenue desc
 limit 20;
+
 ```
+</details>
+
+
+### Feldera Views
+
+Feldera supports three different types of views:
+
+- **Local Views**: **LOCAL** views aren't exposed to the outside world as an
```

Comment:
these are similar to Common Table Expressions in SQL, but we find them easier to use.